org.eclipse.dawnsci.analysis.dataset/src/org/eclipse/dawnsci/analysis/dataset/mask/MaskCircularBuffer.java

```diff
@@ -10,6 +10,7 @@
 import org.eclipse.january.dataset.Comparisons;
 import org.eclipse.january.dataset.Dataset;
 import org.eclipse.january.dataset.DatasetFactory;
+import org.eclipse.january.dataset.IDataset;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.eclipse.january.dataset.IDataset'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)